packages/expo-cli/src/commands/eas-build/status/action.ts

```diff
@@ -7,16 +7,14 @@ import { ensureProjectExistsAsync } from '../../../projects';
 import { printTableJsonArray } from '../../utils/cli-table';
```

Comment:
What's the purpose of the changes in this file? Just refactor?

Reply:
I split the API from the table logging, it's used to query the builds from v2.